jenkins/internal-ci/generic/branch/release.groovy

```diff
@@ -125,15 +125,31 @@ pipeline {
         }
         
         stage ('Sign rpm') {
-            when { expression { false } }
+            when { expression { true } }
             steps {
                 script { build_stage = env.STAGE_NAME }
                                 
                 sh label: 'Generate Key', script: '''
                     set +x
                     pushd scripts/rpm-signing
-                    cat gpgoptions >>  ~/.rpmmacros
-                    sed -i 's/passphrase/'${passphrase}'/g' genkey-batch
+                    for expiredKey in $(gpg2 --list-keys | grep -B1 "Seagate"|head -1|tr -d " ");
+                    do
+                        echo "$expiredKey"
+                        gpg --batch --yes --quiet --delete-secret-key $expiredKey >/dev/null 2>&1
```

Comment:
Do we need `s/gpg/gpg2` here

Reply:
No we don't required gpg2 with gpg also it will work.

Reply:
Should we be consistent to avoid any compatibility issues later

Reply:
ok so you suggest to use gpg2 here right?

I was removed gpg2 from remaining commands as well.

Reply:
If we want to use gpg2 instead of gpg then I will replace it. Can you please confirm?

Reply:
Yes...for consistency use gpg2 at all locations.